NewTutorials/Tutorial/TutorialCasesViewController.swift

```diff
@@ -0,0 +1,99 @@
+//  Copyright © 2017 Oath. All rights reserved.
+
+import UIKit
+import OneMobileSDK
+import PlayerControls
+
+
+class TutorialCasesViewController: UITableViewController {
+    struct Props {
+        var rows: [TextCell.ViewModel] = []
+    }
+    
+    @IBOutlet weak private var activityIndicator: UIActivityIndicatorView!
+    
+    var props: Props = Props() {
+        didSet {
+            guard isViewLoaded else { return }
+            tableView.reloadData()
+        }
+    }
+    
+    func show(viewController: UIViewController) {
+        navigationController?.pushViewController(viewController, animated: true)
+    }
+    
+    func show(error: Error) {
+        let alert = UIAlertController(title: "Error",
+                                      message: "\(error)",
+                                      preferredStyle: .alert)
+        alert.addAction(.init(title: "OK",
+                              style: .default,
+                              handler: nil))
+        present(alert,
+                animated: true,
+                completion: nil)
+    }
+    
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return props.rows.count
+    }
+    
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        guard let cell = tableView.dequeueReusableCell(withIdentifier: "TableCell") as? TextCell else {
+            fatalError("Unknown cell")
+        }
+        cell.viewModel = props.rows[indexPath.row]
+        return cell
+    }
+    
+    override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        props.rows[indexPath.row].action()
+    }
+}
+
+class PlayerViewControllerWrapper: UIViewController {
+    struct Props {
+        var isLoading = false
+    }
+    
+    var props = Props() {
+        didSet {
+            view.setNeedsLayout()
+        }
+    }
+    
+    private let playerViewController: PlayerViewController = {
+        let playerViewController = PlayerViewController()
+        playerViewController.contentControlsViewController = DefaultControlsViewController()
+        return playerViewController
+    }()
+    private let activityIndicatorView: UIActivityIndicatorView = {
+        let activityIndicatorView = UIActivityIndicatorView()
+        activityIndicatorView.activityIndicatorViewStyle = .gray
+        activityIndicatorView.isHidden = true
+        return activityIndicatorView
+    }()
+    
+    func attach(player: Player) {
+        playerViewController.player = player
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        playerViewController.willMove(toParentViewController: self)
```

Comment:
What is the reason of mixing approaches - code and storyboard? Make all required setups in storyboards and add only parts that are not available.